src/Nordea/Components/FilterLabel.elm

```diff
@@ -0,0 +1,67 @@
+module Nordea.Components.FilterLabel exposing (init, view)
+
+import Css
+    exposing
+        ( alignSelf
+        , border3
+        , borderBox
+        , borderRadius
+        , boxSizing
+        , center
+        , cursor
+        , displayFlex
+        , fitContent
+        , flexDirection
+        , fontFamilies
+        , maxWidth
+        , noWrap
+        , padding2
+        , pointer
+        , rem
+        , solid
+        , whiteSpace
+        , width
+        )
+import Html.Styled as Html exposing (Attribute, Html)
+import Html.Styled.Attributes exposing (css)
+import Nordea.Components.Text as Text
+import Nordea.Resources.Colors as Colors
+import Nordea.Resources.Icons as Icon
+import Nordea.Themes as Themes
+
+
+type FilterLabel
+    = FilterLabel { label : String }
+
+
+init : { label : String } -> FilterLabel
+init { label } =
+    FilterLabel { label = label }
+
+
+view : List (Attribute msg) -> FilterLabel -> Html msg
+view attrs (FilterLabel { label }) =
+    Html.button
+        ([ css
+            [ padding2 (rem 0.25) (rem 0.75)
+            , displayFlex
+            , flexDirection Css.row
+            , borderRadius (rem 0.5)
+            , fontFamilies [ "inherit" ]
+            , borderRadius (rem 2)
+            , cursor pointer
+            , boxSizing borderBox
+            , Themes.backgroundColor Themes.SecondaryColor Colors.cloudBlue
+            , Themes.color Themes.PrimaryColor Colors.deepBlue
+            , border3 (rem 0.125) solid Colors.transparent
```

Comment:
Since it is transparent anyway
```suggestion
            , borderWidth (rem 0)
```

Reply:
Good catch! Is it something we have to set at all? 🤔

Reply:
I think _border width 0_ and _appearance none_ are two ways to remove the default styling from buttons. Some browsers prefer one over the other, so having both is common; if I remember correctly.